src/main/webapp/WEB-INF/themes/root/resource/js/pages/peer-review.js

```diff
@@ -24,4 +24,13 @@ $(document).ready(function () {
   if(reviewEl){
     PeerReview.togglePeerReviewAccordion($(reviewEl));
   }
+
+  $('a.supplementary-material__label').each(function(index, item) {
+    var href = $(item).attr('href');
```

Comment:
since `href` isn't used outside of the `if` you could move it in there for clarity.